ci/script/clone_all_rspec_repos

```diff
@@ -9,11 +9,8 @@ if is_mri; then
   clone_repo "rspec-core"
   clone_repo "rspec-expectations"
   clone_repo "rspec-mocks"
-  clone_repo "rspec-rails"
-
-  if rspec_support_compatible; then
-    clone_repo "rspec-support"
-  fi
+  clone_repo "rspec-rails" "main"
```

Comment:
Not all builds (Ruby 2.3, Ruby 2.4) builds are `rspec-rails` compatible.
It's an unnecessary step to check it out for those builds, but I'll leave this improvement for later.